akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectivesSpec.scala

```diff
@@ -162,9 +177,11 @@ class FileAndResourceDirectivesSpec extends RoutingSpec with Inspectors with Ins
       shouldReject("\\")
       shouldReject("%5c")
       shouldReject("..%5c")
-      shouldReject("..%255c")
-      shouldReject("..%c0%af")
-      shouldReject("..%c1%9c")
+
+      // FIXME these don't cause log warnings for some reason
```

Comment:
Perhaps the loglevel?

Reply:
You mean some suspicious strings cause warnings but others cause info logging?